backend_new/src/dtos/shared/id.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsDefined, IsString, IsUUID } from 'class-validator';
+import { Expose } from 'class-transformer';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class IdDTO {
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
+  id: string;
+
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
+  name?: string;
```

Comment:
The swagger docs are getting a bit confused with this on the jurisdiction field. The sample request thinks we need to send the jurisdiction name. It works without sending it. I wonder if there is a way to make the ApiProperty excluded for this. However, I'm guessing your intention is this IdDTO is suppose to be for more than just jurisdiction field. So that might not work
![image](https://github.com/bloom-housing/bloom/assets/42942267/88b1970a-8ebc-45e2-94cc-3f8655fd326c)

Reply:
you're right adding @Apiproperty({ required: false }) fixes the swagger docs for us 

I think I would love if we don't need to add `required: false` to like every field so I'll spend some time figuring out if there is a better way forward

Reply:
@ludtkemorgan , is this comment thread resolved?

Reply:
The issue is still happening, but similar things are happening in the current setup. I have a ticket created to standardize our swagger docs with better sample code and standards. So this can be resolved for this PR